CODEOWNERS

```diff
@@ -1,9 +1,14 @@
+# Misc
+/core/store @samsondav @jmank88
+/core/chains @samsondav
+/core/logger @jmank88
+/core/internal @samsondav @jmank88
+
 # Chains
-/core/chains/evm/headtracker @pinebit @samsondav @prashantkumar1982
```

Comment:
What I did here: remove redundant `/core/chains/evm @samsondav @prashantkumar1982` and move it before `/core/chains/evm/headtracker @pinebit @samsondav @prashantkumar1982` so the latter has priority since it's more specific